packages/rust-bindings/src/aux_functions.rs

```diff
@@ -600,3 +607,80 @@ pub fn deserialize_state_aux(
         Err(e) => Err(anyhow!("Unable to parse state to json: {:?}", e)),
     }
 }
+
+fn get_return_value_schema(
```

Comment:
Will this be moved to contracts-common?

Reply:
That's the plan, yes. The function is part of [this PR](https://github.com/Concordium/concordium-contracts-common/pull/58)